site/docs/master/azure-config.md

```diff
@@ -28,13 +28,15 @@ consider using Premium Managed Disks, which are SSD backed.
 Velero. The tarballs for each release contain the `velero` command-line client. The code in the master branch
 of the Velero repository is under active development and is not guaranteed to be stable!_
 
-1. Extract the tarball:
+2. Extract the tarball:
```

Comment:
If you leave all the numbers in the list as `1.` then it gets auto-numbered when it's rendered; we've tended to prefer this so we don't have to worry about renumbering lists as items get added/removed. Any particular reason you changed it?

Reply:
I'll change it back.

Reply:
Changed.